kong/plugins/aws-lambda/handler.lua

```diff
@@ -0,0 +1,176 @@
+local cjson = require 'cjson'
+local http_client = require 'kong.tools.http_client'
```

Comment:
Okay - I will swap it out when I get a chance to look at this again - which I don't know when that will be...